tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -424,11 +365,86 @@ def assertAllBackendsReceivedRpcs(self, lb_stats):
                 msg=f'Backend {backend} did not receive a single RPC')
 
 
-class TdPropagationRetryableError(Exception):
-    pass
+class XdsKubernetesIsolatedTestCase(XdsKubernetesBaseTestCase,
+                                    metaclass=abc.ABCMeta):
```

Comment:
Recommend to put "Isolated" up front for better readability.

Reply:
Ah, good idea.